plugin/forward/README.md

```diff
@@ -97,6 +97,7 @@ If monitoring is enabled (via the *prometheus* directive) then the following met
 
 * `coredns_forward_request_duration_seconds{to}` - duration per upstream interaction.
 * `coredns_forward_request_count_total{to}` - query count per upstream.
+* `coredns_forward_request_failure_count{to}` - failed query count per upstream.
```

Comment:
This metric name does not fit the naming conventions. I would suggest this:

```
* `coredns_forward_request_failed_total{to}` - failed queries per upstream.
```

Reply:
Thanks for the review @SuperQ. Changed the metric name to `request_failure_count_total` to align with Prometheus naming convention and existing failure counters like `healthcheck_failure_count_total`.

Reply:
Make sense. Will update tomorrow.